package.json

```
@@ -39,6 +39,7 @@
     "expect.js": "0.3",
     "husky": "7.0.4",
     "jsdoc": "3.6.7",
+    "lint-staged": "^12.1.2",
```

```suggestion
    "lint-staged": "12.1.2",
```

Pubrules currently uses the exact version number for all the dependencies.